tutorials/napps/publishing_your_napp.rst

```diff
@@ -179,49 +172,52 @@ Uploading the NApp
 ******************
 
 Your NApp is now ready to be uploaded. 
-Before that, if you do not want to send some files to the server, add them to .gitignore.
-To upload your Napp, use the following command:
+Before that, we need to create a repository with NApp name on github. 
+After creating repository follow the below commands to initialize and push your NApp to github.
 
 .. code-block:: console
 
   $ cd ~/tutorials/<username>/my_first_napp
-  $ kytos napps upload
-  Enter the username: <username>
-  Enter the password for <username>: <password>
-  SUCCESS: NApp <username>/my_first_napp uploaded.
+  $ git init
+  $ git remote add origin https://github.com/<yourGithubAccountName>/<repoName>.git
+  $ git add .
+  $ git push -u origin master
 
-You must fill **username** and **password** with the values you used to
-register your user on the |napps_server|_.
+You will be asked to input your username and password of github, here password indicates generated token from github, View this |github_token|_ to generate your token.
 
-Your NApp is now uploaded. You can see it on the web
-(`<https://napps.kytos.io/<username>/my_first_napp`) or search for it through
-the command line with the following command:
 
-.. code-block:: console
+  Enter username: <username>
+  
+  Enter password: <token>
 
-  $ kytos napps search <username>/my_first_napp
+After finishing the above steps your napp will be pushed to github and your will be able to view it in your repository.
+Follow Instructions in this |github_topic|_ to include topic name 'kytos-ng' to your repository.
```

Comment:
Follow the instructions on...

Reply:
Let's use the topic `kytos-napp` for the NApps instead of `kytos-ng`, we'll use `kytos-ng` only for the platform itself, that way NApps only use `kytos-napp`